lib/json_schemer/schema/base.rb

```diff
@@ -4,17 +4,18 @@ module Schema
     class Base
       include Format
 
-      Instance = Struct.new(:data, :data_pointer, :schema, :schema_pointer, :parent_uri, :before_property_validation, :after_property_validation) do
+      Instance = Struct.new(:data, :data_pointer, :schema, :schema_pointer, :parent_uri, :before_property_validation, :after_property_validation, :regexp_resolver) do
```

Comment:
Does this need to be passed to `Instance` objects? I don't see it being referenced that way